### PR TITLE
fix CAMEL-12149 ensure the CONTENT_TYPE header can be retrieved

### PR DESCRIPTION
**Pull request by xldai**

### CODE REVIEW

File: components/camel-cxf/src/main/java/org/apache/camel/component/cxf/jaxrs/CxfRsProducer.java [hunk 1]
```diff
@@ -164,6 +164,12 @@ protected void invokeAsyncHttpClient(Exchange exchange, final AsyncCallback call
         setupClientMatrix(client, exchange);
         setupClientQueryAndHeaders(client, exchange);
 
+        // ensure the CONTENT_TYPE header can be retrieved
```
Comment (davsclaus) [inline components/camel-cxf/src/main/java/org/apache/camel/component/cxf/jaxrs/CxfRsProducer.java]: We have the `null` on the other side as out standard with the Apache Camel code. Do you mind flipping them around.

**Comment by oscerd** [comment]
Can't we use ObjectHelper?

**Comment by davsclaus** [comment]
Yeah ObjectHelper.isEmpty would also check for an empty string, however not sure if you would have a Content-Type header that would be a `""`.

**Comment by xldai** [comment]
Claus, I made minor changes by using ObjectHelper, is that OK? or other concerns you may have?

**Pull request merged**